nestjs/src/courses/score/dto/GetScoreQuery.dto.ts

```diff
@@ -0,0 +1,60 @@
+import { ApiProperty, ApiPropertyOptional } from '@nestjs/swagger';
```

Comment:
please use `kebab-case` notation for files as we do everywhere for nest.js

Reply:
renamed, thanks